documentation/build.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+import os
+
+
+def run():
+    """
+    Run all unittests. Equivalent to:
```

Comment:
Not sure if it should say `unittests` here.

Reply:
you are right